Bottom: b6ddae78fb2a5554ec4ab5a119a072dc28875c21
Top:    5b4f7903191312ad9df2eb7674984050cb644c69
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:10:04 -0500

Refresh of bacula-update-syntax

---
diff --git a/roles/mariadb/tasks/main.yml b/roles/mariadb/tasks/main.yml
index 966c9a93..dc0a5b6d 100644
--- a/roles/mariadb/tasks/main.yml
+++ b/roles/mariadb/tasks/main.yml
@@ -3,22 +3,36 @@
     that:
     - mariadb_root_password is defined
 - name: Install MariaDB
-  yum: name={{ item }} state=present
+  yum: 
+    name: '{{ item }}'
+    state: present
   with_items:
     - mariadb-server
     - mariadb
 - name: Config MariaDB server
-  template: src=ansible.cnf.j2 dest={{ mariadb_server_config_dir }}/ansible.cnf
-            owner=root group=root mode=0644 backup=yes
+  template:
+    src: ansible.cnf.j2
+    dest: '{{ mariadb_server_config_dir }}/ansible.cnf'
+    owner: root 
+    group: root 
+    mode: 0644 
+    backup: yes
 - name: Enable and start MariaDB service
-  service: name=mariadb enabled=yes state=started
+  service: 
+    name: mariadb 
+    enabled: yes 
+    state: started
 - name: Check previous installation
-  stat: path={{ mariadb_client_config }}
+  stat: 
+    path: '{{ mariadb_client_config }}'
   register: cfg
 - include_tasks: initial_config.yml
   when: cfg.stat.exists is defined and cfg.stat.exists == false
 - name: Open the firewall
-  firewalld: service=mysql permanent=true state=enabled
+  firewalld: 
+    service: mysql 
+    permanent: true 
+    state: enabled
   when: mariadb_open_firewall
 - name: Set host mysqld bind-address
   template:
